osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/infra/networking/services/commands/Iec61850SetScheduleCommand.java

```diff
@@ -77,7 +82,7 @@ public void setScheduleOnDevice(final Iec61850Client iec61850Client, final Devic
         try {
             // Creating a list of all Schedule entries, grouped by relay index.
             final Map<Integer, List<ScheduleEntry>> relaySchedulesEntries = this.createScheduleEntries(scheduleList,
-                    ssld, relayType, ssldDataService);
+                    ssld, relayType);
 
             final Function<Void> function = new Function<Void>() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this anonymous class number of lines from 164 to at most 20, or make it a named class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1188)